Source/Scene/TranslucentTileClassification.js

```diff
@@ -199,18 +150,9 @@ function updateResources(
 
   var width = transpClass._opaqueDepthStencilTexture.width;
   var height = transpClass._opaqueDepthStencilTexture.height;
-
-  var colorTexture = transpClass._colorTexture;
-  var textureChanged =
-    !defined(colorTexture) ||
-    colorTexture.width !== width ||
-    colorTexture.height !== height;
-  if (textureChanged) {
+  if (transpClass._drawClassificationFBO.isDirty(width, height)) {
```

Comment:
I think this check can also be removed because `drawClassificationFBO`'s color texture is not shared. Is it possible for only some of the framebuffers to be dirty here?

Reply:
As of 7917da9 it looks like this check is still needed, I'll have to dig into this a bit more.